wallet/psbt_test.go

```diff
@@ -245,7 +386,7 @@ func TestFundPsbt(t *testing.T) {
 		t.Run(tc.name, func(t *testing.T) {
 			changeIndex, err := w.FundPsbt(
 				tc.packet, nil, 1, 0, tc.feeRateSatPerKB,
-				CoinSelectionLargest,
+				CoinSelectionLargest, WithCustomChangeScope(tc.changeKeyScope),
```

Comment:
Perhaps don't pass a modifier at all if `tc.changeKeyScope` is nil, because in real use it's probably not set either? It doesn't matter too much, because the modifier is a no op with nil.

Reply:
Up to you

Reply:
IMO, I would let like it is as it is a no-op (and we avoid a `if` right before)

+ `WithCustomChangeScope(nil)` and `nil` use the same path in the code